Documentation/ApiOverview/Events/Hooks/Index.rst

```diff
@@ -85,70 +83,70 @@ The syntax of a function reference can be seen in the API documentation of
 Creating hooks
 ==============
 
-You are encouraged to create hooks in your extensions if they seem
-meaningful. Typically someone would request a hook somewhere. Before
-you implement it, consider if it is the right place to put it. On
-the one hand we want to have many hooks but not more than needed.
-Redundant hooks or hooks which are implemented in the wrong context is
-just confusing. So put a little thought into it first, but be
-generous.
+..  note::
+    It is highly recommended to dispatch `Psr-14 events
+    <EventDispatcherQuickStartDispatching>` instead of introducing new hooks.
+    Existing hooks should be migrated to events.
 
 There are two main methods of calling a user defined function in
 TYPO3.
 
-- :code:`\TYPO3\CMS\Core\Utility\GeneralUtility::callUserFunction()` - The classic way. Takes a
-  file/class/method reference as value and calls that function. The
-  argument list is fixed to a parameter array and a parent object. So
-  this is the limitation. The freedom is that the reference defines the
-  function name to call. This method is mostly useful for small-scale
-  hooks in the sources.
-
-- :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance()` - Create an
-  object from a user defined
-  file/class. The method called in the object is fixed by the hook, so
-  this is the non-flexible part. But it is cleaner in other ways, in
-  particular that you can even call many methods in the object and you
-  can pass an arbitrary argument list which makes the API cleaner.
-  You can also define the objects to be singletons,
-  instantiated only once in the global scope.
+:php:`\TYPO3\CMS\Core\Utility\GeneralUtility::callUserFunction()`
+    The classic way. Takes a
+    file/class/method reference as value and calls that function. The
+    argument list is fixed to a parameter array and a parent object. So
+    this is the limitation. The freedom is that the reference defines the
+    function name to call. This method is mostly useful for small-scale
+    hooks in the sources.
+
+:php:`\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance()`
+    Create an object from a user defined
+    file/class. The method called in the object is fixed by the hook, so
+    this is the non-flexible part. But it is cleaner in other ways, in
```

Comment:
Two times "cleaner" in the sentence. Can we avoid it?

"But it is cleaner ... makes the API cleaner".